admin_ui/src/components/RowFormSelect.vue

```diff
@@ -24,40 +27,29 @@
                     >
                         <font-awesome-icon icon="plus" />
                     </router-link>
-
-                    <router-link
-                        :to="{
-                            name: 'editRow',
-                            params: {
-                                tableName: property.extra.to,
-                                rowID: getKeySelectID(property.title),
-                            },
-                        }"
-                        class="add"
-                        target="_blank"
-                        v-if="!isFilter"
-                    >
-                        <font-awesome-icon icon="edit" />
-                    </router-link>
```

Comment:
Do we definitely want to remove this?

Reply:
No, we keep this. I probably accidentally deleted this.

Reply:
For now I remove this because  to of `rowID` is undefined in `ReferencingTables.vue`. I think that problem is [here](https://github.com/piccolo-orm/piccolo_admin/blob/master/admin_ui/src/components/RowFormSelect.vue#L118). I tried `Reflect.set()` and `this.keySelectIDs[propertyTitle] = value` but neither way works.

Reply:
This feature returns with commit [7364542](https://github.com/piccolo-orm/piccolo_admin/pull/89/commits/736454244de6cfc006d16d766ab016a61b9613f2).